scripts/write-ver-to-manifest.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import { execSync } from 'child_process';
+import viteConfig from '../vite.config';
+import dotenv from 'dotenv';
+
+// Load environment variables
+dotenv.config();
+
+const { build } = viteConfig;
+
+if (!build || !build.outDir) {
+  throw new Error('Build output directory is not defined in vite.config.ts');
+}
+const manifestPath = `./${build?.outDir}/.vite/manifest.json`;
+
+const manifest = fs.readFileSync(manifestPath, 'utf-8');
+
+const manifestJson = JSON.parse(manifest);
+
+// Get current commit hash
+const hash =
```

Comment:
@francisaguilar21 Just added this one to prioritize getting version from env per @guerzon's suggestion. Then if git exists try to set it to the head commit.

Please let us know if `HEAD_COMMIT_HASH` is something we could add to the build environment @jasontorres.